prov/tcp/src/tcpx_attr.c

```diff
@@ -80,7 +80,9 @@ static struct fi_fabric_attr tcpx_fabric_attr = {
 };
 
 struct fi_info tcpx_info = {
-	.caps = FI_MSG | FI_SEND | FI_RECV,
+	.caps = FI_MSG | FI_SEND | FI_RECV |
+	FI_RMA | FI_WRITE | FI_REMOTE_READ |
```

Comment:
Commit message only calls out rma write.  Is setting FI_REMOTE_READ correct?  If not, it should be removed from this patch.  If it is, the commit message should be updated.  And please provide more details in the commit messages than a single line! :)